Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -130,15 +130,19 @@ function Initialize-AzureSubscription {
         } else {
             # Else, this is AzureRM.
             try {
-                if(CmdletHasMember -cmdlet "Add-AzureRMAccount" -memberName "EnvironmentName")
-                {
-                    Write-Host "##[command]Add-AzureRMAccount -ServicePrincipal -Tenant $($Endpoint.Auth.Parameters.TenantId) -Credential $psCredential -EnvironmentName $environmentName"
-                    $null = Add-AzureRMAccount -ServicePrincipal -Tenant $Endpoint.Auth.Parameters.TenantId -Credential $psCredential -EnvironmentName $environmentName
+                if (Get-Command -Name "Add-AzureRmAccount" -ErrorAction "SilentlyContinue") {
```

Comment:
Should you update in releases/M131 so that earlier version of task also get this?

Reply:
done